packages/cadl-ranch-specs/http/resiliency/cadl-project.yaml

```diff
@@ -0,0 +1,2 @@
+emitters:
```

Comment:
Should I commit this file?

Reply:
I think we should only need one `cadl-project.yaml` at the root if we want to configure the default emitter. SHould it be `cadl-autorest` though for now at least?